commands/backfill.js

```diff
@@ -12,6 +14,8 @@ module.exports = function (program, conf) {
     .option('-d, --days <days>', 'number of days to acquire (default: ' + conf.days + ')', Number, conf.days)
     .option('--start <unix_in_ms>', 'lower bound as unix time in ms', Number, -1)
     .option('--end <unix_in_ms>', 'upper bound as unix time in ms', Number, -1)
+    .option('--start_date <datetime>', 'start date in format ("YYYYMMDDhhmm")', Number, -1)
+    .option('--end_date <datetime>', 'end date in format ("YYYYMMDDhhmm")', Number, -1)
```

Comment:
can we support both unix timestamp and datetime stamp for the same parameter?

Reply:
i have no idea, let me check moment docs to see if there is a way to diffrentiate them